.gitignore

```diff
@@ -118,3 +118,6 @@ env/
 
 ### Configuration files ###
 .env
+
+### geckodriver log
+geckodriver.log
```

Comment:
Good spot but note that it's already been committed! If you add to `.gitignore` after staging the file, you'll need to separately remove it from Git with `git rm --cached geckodriver.log`